### PR TITLE
docs: Update next param ts def

### PR DESCRIPTION
**Pull request by zombieJ**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: plugin/controller/README.md [hunk 1]
```diff
@@ -49,12 +49,12 @@ Middleware 支持多个入参，依次传入要生效的中间件
 // app/middleware/global_log.ts
 import { Context } from 'egg';
 
-export default async function globalLog(ctx: Context, next: any) {
+export default async function globalLog(ctx: Context, next: VoidFunction) {
```
Comment (killagu) [inline plugin/controller/README.md:52]: 有个 Next 的定义的。

Comment (zombieJ) [reply plugin/controller/README.md:52]: OK 我改改

**Review by killagu — APPROVED** [review]
LGTM

File: plugin/controller/README.md [hunk 2]
```diff
@@ -48,13 +48,14 @@ Middleware 支持多个入参，依次传入要生效的中间件
 ```ts
 // app/middleware/global_log.ts
 import { Context } from 'egg';
+import type { Next } from '@eggjs/controller-decorator';
```
Comment (fengmk2) [inline plugin/controller/README.md:51]: import type { Next } from '@eggjs/tegg;

应该都改成从 tegg 导出了吧。

**Pull request merged**